extension/src/test/e2e/util.ts

```diff
@@ -106,6 +107,42 @@ export const waitForViewContainerToLoad = async (): Promise<void> => {
   )
 }
 
+export const deleteAllExistingExperiments = async () => {
```

Comment:
[F] I think this was one of the causes of local instability. We were not starting from the same base on every run.